Add 1 on 1 video calling/index.html

```diff
@@ -0,0 +1,48 @@
+<!DOCTYPE html>
```

Comment:
nit: since the change contains UI change, if we could attach a picture to show the UI final look that would be great!

Reply:
Picture attached in the linked document.